.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -27,15 +27,27 @@ jobs:
     name: Azure-Pipelines-CI-Test-EO
   steps:
   - task: ComponentGovernanceComponentDetection@0
-
+  ## dev builds are in format Major.Minor.Patch-CommitAheadCount-CommitId
+  ## prod release are in format Major.Minor.Patch
+  ## preview release are in format preview-Major.Minor.Patch
   - bash: |
-      commit=$(git rev-parse --short HEAD)
-      datetime=$(date +'%m%d%Y')
-      linuxImagetag="$datetime"-"$commit"
-      windowsImageTag=win-"$datetime"-"$commit"
+      commit=$(git describe) 
+      linuxImagetag="$commit"
+      windowsImageTag=win-"$commit"
+
+      tagConnector="-"
```

Comment:
not really clear here? it looks like telemetryTag is same as prefix? is there a scenario where it's not the same?

Reply:
eg assuming commit is 3.1.1-22-g
linuxImageTag = 3.1.1-22-g

prefix = 3.1.1
index = 5
telemetryTag = 3.1.1

therefore, prefix is same as telemetryTag? then why the extra steps? am I missing something

Reply:
addressed https://github.com/microsoft/Docker-Provider/pull/876/commits/7265fe9609f5643347a19d0c2ee2802ba880269d